mentat/code_feature.py

```diff
@@ -55,10 +55,10 @@ def split_file_into_intervals(
                 )
         else:
             line_number = 1
-        if i == len(ctags) - 1:
-            named_intervals.append((str(key), int(line_number), n_lines))
+        if i == len(ctags) - 1:  # type: ignore
+            named_intervals.append((str(key), int(line_number), n_lines))  # type: ignore
         else:
-            _last_item = (key, int(line_number))
+            _last_item = (key, int(line_number))  # type: ignore
```

Comment:
These pyright errors might already be fixed in the parent branch

Reply:
Since we'll be merging in to main anyways, it might be best to remove the type ignores now so we don't forget later? Up to you